doc/faq.rst

```diff
@@ -1,3 +1,20 @@
+.. raw:: html
+
+  <!-- h3 headings on this page are the questions; make them rubric-like -->
+  <style>
+    h3 {
+      font-size: 1rem;
+      font-weight: bold;
+      padding-bottom: 0.2rem;
+      margin: 2rem 0 1.15rem 0;
+      border-bottom: 1px solid var(--pst-color-border);
+    }
+
+    h2 + section > h3 {
+      margin-top: 2.5rem;  /* Increase top margin for first question in each section */
+    }
+  </style>
```

Comment:
I agree however that I don't like the fact that the headers look like backlinks back to the top level ToC:

![image](https://github.com/scikit-learn/scikit-learn/assets/89061/6fbccb02-d303-44f9-b179-e76d6d2f5aa9)

Both because of the link style of the headers and because they scroll back to an invisible location of the ToC because of the fixed nav bar. Maybe that later point could be fixed somehow, but I am fine just removing the ToC backlinks on this page.